### PR TITLE
SMIME stderr bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-19569)

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.py [hunk 1]
```diff
@@ -73,6 +76,8 @@ def encrypt_email_body(client: Client, args: Dict):
         args: Dict
 
     """
+    from M2Crypto import BIO, SMIME, X509, m2
+    print("TEST")
```
Comment (ShahafBenYakir) [inline Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.py:80]: Is this for testing? Why are we importing this twice?

File: Packs/SMIME_Messaging/ReleaseNotes/1_0_6.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### SMIME Messaging
+Fixed an issue where unwanted messages from external python libraries were printed to screen.
```
Comment (ShahafBenYakir) [inline Packs/SMIME_Messaging/ReleaseNotes/1_0_6.md:4]:
```suggestion
Fixed an issue where unrelated messages from external python libraries were printed to the screen.
```

**Review by ostolero — CHANGES_REQUESTED** [review]
Hi Dan, Please add the `EWS Mail Sender Test 3` TPB under the integrations yml `tests` section.
As it is configured in conf.json it should run here as well.
Also this is a nightly test so let's make sure it passes in build before merging. 
Let me know if you need any help with the TPB, tnx

**Review by ostolero — COMMENTED** [review]
Looks great! did you get a chance to check if a unit test could be added to detect this issue in the future?

**Review by ostolero — APPROVED** [review]
Looks great 👍

**Pull request merged**